frontend/components/SlideCard.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
+    <ComponentCard :badge="idx" title-class="">
+      <template #title>
+        <code>{{ comment.path }}</code>
+      </template>
+      <template #body>
+        <div class="overflow-scroll text-lg max-h-[70vh]">
+          <DiffBlock :comment="comment" />
+        </div>
+      </template>
+    </ComponentCard>
+    <div class="max-w-2xl mx-auto text-center">
+      <MarkdownHTML>{{ commentBody }}</MarkdownHTML>
+    </div>
+  </div>
+</template>
```

Comment:
3. 

`SlideCard` is a new component for how we render slides. 

Currently, it goes basically full width, reusing the `ComponentCard` which is a horrible name, for the code-block-diff and then the comment underneath (no real styling).